build.gradle

```diff
@@ -63,6 +63,7 @@ configurations
         // Hadoop and Spark are way too fat.
         exclude group: 'org.apache.hadoop'
         exclude group: 'org.apache.spark'
+        exclude group: 'org.slf4j', module: 'slf4j-api'
```

Comment:
Is this OK to add here or does this change too much. The reason for this change was that when the logger was included in the shaded build it broke when running it on AWS.